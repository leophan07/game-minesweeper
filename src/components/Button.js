import styled from 'styled-components';

export default styled.button`
  padding: 5px 20px;
  cursor: pointer;
  border-radius: 5px;
`;