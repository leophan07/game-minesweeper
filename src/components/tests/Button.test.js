import React from 'react';
import { render } from '@testing-library/react';

import Button from '../Button';

describe('<Button />', () => {
  it('should render an <button> tag', () => {
    const { container } = render(<Button />);
    expect(container.querySelector('button')).toBeTruthy();
  });

  it('Button should match the snapshot', () => {
    const { container } = render(<Button />);
    expect(container).toMatchSnapshot();
  });
});
