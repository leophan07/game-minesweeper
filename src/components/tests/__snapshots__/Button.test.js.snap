// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> Button should match the snapshot 1`] = `
<div>
  <button
    class="sc-bdVaJa gCtIyx"
  />
</div>
`;
